### PR TITLE
feat(k8s): add routed ip migration dates

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by Lawndlwd — APPROVED** [review]

**Review by jcirinosclwy — APPROVED** [review]

**Review by jsudraud — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

File: containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx [hunk 1]
```diff
@@ -25,8 +25,11 @@ Additionally, routed IPs unlock the ability to attach multiple IPv4 addresses to
 </Message>
 
 <Message type="important">
-    NAT IPs will reach end-of-life on October 1st 2024. After this date, Scaleway will organize migrations for all clusters to transition to routed IPs.
-    Migrations will be planned region by region, with specific dates to be announced.
+    NAT IPs will reach end-of-life on October 1st, 2024. After this date, Scaleway will organize migrations for all clusters to transition to routed IPs.
+    Migrations are planned region by region:
+    * For clusters in Warsaw (`pl-waw-1`), the migration is scheduled for Oct 1 to Oct 7, 2024. [Learn more](https://status.scaleway.com/incidents/l45384bsm5w0).
```
Comment (nerda-codes) [inline containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx:30]:
```suggestion
    * For clusters in Warsaw (`pl-waw-1`), the migration is scheduled for October 1st to October 7th, 2024. [Learn more](https://status.scaleway.com/incidents/l45384bsm5w0).
```

Comment (nerda-codes) [reply containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx:30]: same for others

File: containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx [hunk 2]
```diff
@@ -25,8 +25,11 @@ Additionally, routed IPs unlock the ability to attach multiple IPv4 addresses to
...
+    * For clusters in Amsterdam (`nl-ams-1`, `nl-ams-2`, and `nl-ams-3`), the migration is scheduled for Oct 7 to Oct 14, 2024. [Learn more](https://status.scaleway.com/incidents/bctz2b9fhpqc).
```
Comment (bene2k1) [inline containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx:31]:
```suggestion
    * For clusters in Amsterdam (`nl-ams-1`, `nl-ams-2`, and `nl-ams-3`), the migration is scheduled for October 7 to October 14, 2024. [Learn more](https://status.scaleway.com/incidents/bctz2b9fhpqc).
```

File: containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx [hunk 3]
```diff
@@ -25,8 +25,11 @@ Additionally, routed IPs unlock the ability to attach multiple IPv4 addresses to
...
+    * For clusters in Paris (`fr-par-1`, `fr-par-2`, and `fr-ar-3`), the migration is scheduled for Oct 14 to Oct 21, 2024. [Learn more](https://status.scaleway.com/incidents/1gh354wb3j27).
```
Comment (bene2k1) [inline containers/kubernetes/reference-content/move-kubernetes-nodes-routed-ip.mdx:32]:
```suggestion
    * For clusters in Paris (`fr-par-1`, `fr-par-2`, and `fr-ar-3`), the migration is scheduled for October 14 to October 21, 2024. [Learn more](https://status.scaleway.com/incidents/1gh354wb3j27).
```

**Pull request merged**